osf/external/askismet/client.py

```diff
@@ -133,3 +133,31 @@ def submit_ham(self, user_ip, user_agent, **kwargs):
         )
         if res.status_code != requests.codes.ok:
             raise AkismetClientError(reason=res.text)
+
+    def get_flagged_count(self, start_date, end_date, category='node'):
+        from osf.models import NodeLog, PreprintLog
+
+        log_model = NodeLog if category == 'node' else PreprintLog
```

Comment:
Minor: it would be preferable if an error was raised if `category` wasn't in an controlled list of terms (e.g. `['node', 'preprint']`)